### ISSUE TITLE
ratatui name is already taken

### ISSUE THREAD

**Issue by rbtcollins** [tui-rs-revival/ratatui#83]
https://crates.io/search?q=ratatui - perhaps thats a project member from here, but if not, maybe another name change is needed.

**Comment by AntoniosBarotsis** [comment]
The crate owners (or at least 2 of them) are contributors to this project, so I'd imagine the placeholder was from them.

**Comment by Cyborus04** [comment]
Yes, that was to claim the name early - the owner of the crate said so here https://discord.com/channels/1070692720437383208/1072889040883417209/1072926306968293446

For those not on discord:
> mrjackwills: should someone grab the name, just in case?

> Mindoo: I'm unfamiliar with the process, if you feel comfortable doing so, sure go ahead. Make sure you "release" it if we don't end up using it though.

> mrjackwills: yeah, I'll grab it, and then if we vote to use something else, I'll release it
